### PR TITLE
Add Danish translations

### PR DESCRIPTION
**Pull request by DuckoDas**
**Describe Pull request:**
Ive added the da.lua file for the Danish qbcore users that want their server to be Danish instead of English or other languages

**If your PR is to fix an issue mention that issue here:**
I just added the Danish language to QB-Policejob

**Questions (please complete the following information):**

**Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest):** Yes cause i have a Danish server
**Does your code fit the style guidelines? [yes/no]:** Yes
**Does your PR fit the contribution guidelines? [yes/no]:** Yes

### CODE REVIEW

File: locales/da.lua [hunk 1]
```diff
@@ -0,0 +1,195 @@
+local Translations = {
+    error = {
+        license_already = 'Spilleren har allerede en licens',
+        error_license = 'Afspilleren har ikke den licens',
+        no_camera = 'Kamera eksisterer ikke',
+        blood_not_cleared = 'Blod IKKE renset',
+        bullet_casing_not_removed = 'Kuglehuse IKKE fjernet',
+        none_nearby = 'Ingen i nærheden!',
+        annulleret = 'Annulleret..',
```
Comment (i-kulgu) [inline locales/da.lua:9]: annulleret doesn't exist in language files.

File: locales/da.lua [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    success = {
+        uncuffed = 'Du er blevet uncuffed',
+        granted_license = 'Du har fået en licens',
+        grant_license = 'Du har givet en licens',
+        revoke_license = 'Du tilbagekaldte en licens',
+        tow_paid = 'Du blev betalt 500 kr.',
+        blood_clear = 'Blod renset',
+        bullet_casing_removed = 'Kuglehuse fjernet...',
+        anklet_taken_off = 'Din ankeltracker er taget af.',
+        Take_anklet_from = 'Du tog %{firstname} %{lastname} tracker',
```
Comment (i-kulgu) [inline locales/da.lua:40]: Take_anklet_from must be : took_anklet_from

File: locales/da.lua [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        put_anklet = 'Du tager en ankelsporer på.',
+        put_anklet_on = 'Du sætter en ankeltracker på %{firstname} %{lastname}',
+        vehicle_flagged = 'Køretøj %{plade} er blevet markeret for %{reason}',
```
Comment (i-kulgu) [inline locales/da.lua:43]: %{plade} needs to be %{plate}

File: locales/da.lua [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        impound_vehicle_removed = 'Køretøj taget ud af beslaglæggelse!',
+        beslaglagt = 'Køretøj beslaglagt',
```
Comment (i-kulgu) [inline locales/da.lua:45]: beslaglagt doesn't exist...

File: locales/da.lua [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+ },
+    info = {
+        mr = 'Hr.',
+        fru = 'Fru',
```
Comment (i-kulgu) [inline locales/da.lua:49]: fru doesn't exist....

File: locales/da.lua [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+        fine_received = 'Du har modtaget en bøde på %{fine} kr.',
+        blip_text = 'Politialarm - %{value}',
+        jail_time_input = 'Fængselstid',
+        submit = 'Indsend',
+        time_months = 'Tid i måneder',
+        bill = 'Bøde',
+        amount = 'Beløb',
+        police_plate = 'DKPL', --Bør kun være 4 tegn langt
+        vehicle_info = 'Motor: %{value} % | Brændstof: %{value2} %',
+        evidence_stash = 'Bevis Stash | %{værdi}',
```
Comment (i-kulgu) [inline locales/da.lua]: %{værdi} ?

File: locales/da.lua [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+        slot = 'Slot nr. (1,2,3)',
+        current_evidence = '%{værdi} | Skuffe %{value2}',
```
Comment (i-kulgu) [inline locales/da.lua]: %{værdi} ?

File: locales/da.lua [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+        on_duty = '[E] - Gå på vagt',
+        off_duty = '[E] - Tag fri',
+        onoff_duty = '~g~On~s~/~r~Off~s~ Duty',
+        stash = 'Stash %{value}',
+        delete_spike = '[~r~E~s~] Slet Spike Strip',
+        close_camera = 'Luk kamera',
+        bullet_casing = '[~g~G~s~] Bullet casing %{value}',
+        casing = 'Bullet casing',
+        blod = 'Blod',
```
Comment (i-kulgu) [inline locales/da.lua:98]: blod doesnt exist ...

File: locales/da.lua [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+        blood_text = '[~g~G~s~] Blod %{value}',
+        fingerprint_text = '[G] Fingeraftryk',
+        fingeraftryk = 'Fingeraftryk',
```
Comment (i-kulgu) [inline locales/da.lua:101]: fingeraftryk doesn't exist...

File: locales/da.lua [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+        message = 'Kære %{value}. %{value2}, <br /><br />Central Judicial Collection Agency (CJCA) opkrævede de bøder, du modtog fra politiet.<br />Der er <strong>%{value3} kr.</strong> trukket fra din konto.<br /><br />Med venlig hilsen<br />Hr. I.K. Graai',
+    },
+    commands = {
+        place_spike = 'Placer Spike Strip (kun politiet)',
+        license_grant = 'Giv en licens til nogen',
+        license_revoke = 'Tilbagekald en licens fra nogen',
+        place_object = 'Placer/slet et objekt (kun politiet)',
+        cuff_player = 'Sæt håndjern på spiller (kun politiet)',
+        escort = 'Eskortespiller',
+        kaldesignal = 'Giv dig selv et kaldesignal',
```
Comment (i-kulgu) [inline locales/da.lua]: kaldesignal doesn't exist..

File: locales/da.lua [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+        clear_casign = 'Ryd område for hylstre (kun politiet)',
+        jail_player = 'Fængselsspiller (kun politiet)',
+        unjail_player = 'Fjern spilleren (kun politiet)',
+        clearblood = 'Ryd området for blod (kun politiet)',
+        seizecash = 'Beslaglæg kontanter (kun politiet)',
+        softcuff = 'Sæt Bløde Håndjern på spiller (kun politiet)',
+        camera = 'Se sikkerhedskamera (kun politiet)',
+        flagplade = 'Flag en plade (kun politiet)',
```
Comment (i-kulgu) [inline locales/da.lua]: flagplade should be flagplate

Comment (DuckoDas) [reply locales/da.lua]: its danish so no

Comment (i-kulgu) [reply locales/da.lua]: > its danish so no

The translation may be in danish but the word is :  flagplate = ...... , so yes

**Review by i-kulgu — APPROVED** [review]
LGTM

**Review by zwkf — APPROVED** [review]
LGTM

**Pull request merged**